ruby/jukebox/jukebox.rb

```diff
@@ -125,3 +125,6 @@ def choose_direction(usage)
     puts "Hmm. No matches found. Try adding someone who fits that description!"

```

Comment:
Please remove all gets.chomp for methods.  gets.chomp should only be for user input inside driver code. gets.chomp should not be included in methods.